docs/PROVIDER.md

```diff
@@ -115,4 +115,57 @@ rack1 = {
 ~~~
 
 Terraform will always use the most specific client values. In the case client credentials are defined at both the provider block and resource level, **the credentials defined at the resource level** will be used. 
-  
\ No newline at end of file
+
+## 2. Important limitation regarding resource creation
+Normally Terraform is able to create resources in a parallel manner. With this provider we face some challenges, because for the great majority of resource creation/update, the servers need to be rebooted. This means that, for the provider to work as expected when creating several resources, they MUST be created one by one (secuentially).  
+
+At the moment of writing this, Terraform core does not allow secuential resource creation, so in case operators want to create more than one resource at a time (which would be a normal scenario), they'd need to use the **depends_on** tag, creating a dependancy with the previous resource. Taka a look into the following code snippet:
```

Comment:
```suggestion
At the moment of writing this, Terraform core does not allow sequential resource creation, so in case operators want to create more than one resource at a time (which would be a normal scenario), they'd need to use the **depends_on** tag, creating a dependency with the previous resource. Taka a look into the following code snippet:
```